javascript/javascript1/week4/getReply.js

```diff
@@ -0,0 +1,103 @@
+let userName = " ";
+let todo = [];
+function getReply(command) {
+  //'ask for name.'
+  const array = command.split(" ");
+  let response = " ";
+  if (command.includes("hello my name is ")) {
+    userName = array[array.length - 1];
+    return "Nice to meet you " + userName;
+  } else if (command.includes("what is my name")) {
+    return "your name is " + userName;
+  } else if (command.includes() {
+    return "your name is not find";
+  }
+  //'adding to todos.'
+  else if (command.includes("add ") && command.includes(" to my todos")) {
+    todo.push("fishing");
+    todo = command(command.length - 1);
```

Comment:
As of the current syntax you are trying to set todo's value to the `command` function's returned value.
But since there is no `command` function declared in your code, I am suspecting that you'd like to do something different here. If you tell me what your intention is here, I can help you with it.

Reply:
else if (commandLowerCase.includes("add ") && commandLowerCase.includes(" to my todos")) {
    todo = command[command.length - 1];
    todo.push("fishing");
    return "fishing is added to your todos";
  } else if (commandLowerCase.includes("add singing")) {
     todo.push("singing");
     return "singing is added to your todos";
  } else if (commandLowerCase.includes("remove fishing")) {
    for (let i = 0; i < todo.length; i++) {
      if (todo[i].includes("fishing")) {
        todo.splice(i, 1);
        return "removed fishing from your todos";
      }
    }
  }